packages/amplify-e2e-tests/src/environment/env.ts

```diff
@@ -20,6 +20,28 @@ export function addEnvironment(cwd: string, settings: { envName: string; numLaye
   });
 }
 
+export function addEnvironmentCarryOverEnvVars(cwd: string, settings: { envName: string }): Promise<void> {
```

Comment:
Please make this an async function and use the newer `.runAsync()` method at the end of the chain returned by `spawn(...)`